src/shared/constants/getNextPage.constants.ts

```diff
@@ -0,0 +1,8 @@
+/**
+ * If delegated examiner test nav directly to office page instead of debrief page
+ * @param category
+ * @param delegatedTest
+ */
+export function getNextPage(category, delegatedTest) {
+  return delegatedTest ? category.OFFICE_PAGE : category.DEBRIEF_PAGE;
+}
```

Comment:
feel like this function/file should be named something more specific, currently sounds like its some kind of common func which could be used anywhere to get the next page rather than the page after test report, other than that looks good

Reply:
yes agree tom, needs to mention delegated specifically

Reply:
renamed file and function.